python/paddle/optimizer/radam.py

```diff
@@ -133,15 +152,17 @@ class RAdam(Optimizer):
 
     def __init__(
         self,
-        learning_rate=0.001,
-        beta1=0.9,
-        beta2=0.999,
-        epsilon=1.0e-8,
-        parameters=None,
-        weight_decay=None,
-        grad_clip=None,
-        name=None,
-    ):
+        learning_rate: float | LRScheduler = 0.001,
+        beta1: float | Tensor = 0.9,
+        beta2: float | Tensor = 0.999,
+        epsilon: float = 1.0e-8,
+        parameters: Sequence[Tensor]
+        | Sequence[_RAdamParameterConfig]
+        | None = None,
```

Comment:
```suggestion
        parameters: (
            Sequence[Tensor] | Sequence[_RAdamParameterConfig] | None
        ) = None,
```

可以试一下这个，我在 3.10 测试没问题，如果 3.8 也没问题的话，就推广下这个代码风格吧，其它优化器和类似的地方都可以改一下～